### ISSUE TITLE
Keywords

### ISSUE THREAD

**Issue by xogeny** [impact/impact#104]
I think it would be good to add a `keywords` field to the index entry for each library.  This could be used to provide additional "search" terms.  The question then is, how would this `keywords` field be populated.  At a minimum, I think the same field then should be added to the `dirinfo` structure as well as a means for library developers to provide keywords.  We might also consider leveraging an annotation in the `package.mo` file.

**Comment by xogeny** [comment]
Thinking about this, I think we should have both keywords (human readable) for things like "open-source" or other "tag" like attributes **and** a `search_terms` entry to provide affinity to certain search terms.  This `search_term` entry could be populated with, for example, things like "clutch" (*i.e.,* terms that match the contents).  The idea is that you wouldn't want to list all possible search terms in a listing about the library, but you could list keywords.

**Comment by tbeu** [comment]
Could the impact.json file be (re)used here?

**Comment by xogeny** [comment]
Yes, that is exactly what I had in mind when I talked about the `dirinfo` information.  That is what is constructed [when parsing the `impact.json` file](https://github.com/impact/impact/blob/master/crawl/heuristics.go#L166).
